SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -563,10 +563,21 @@ void USpatialNetDriver::BeginDestroy()
 {
 	Super::BeginDestroy();
 
-	// If we are still connected, cleanup our corresponding worker entity if it exists.
+	// If we are still connected...
 	if (Connection != nullptr && WorkerEntityId != SpatialConstants::INVALID_ENTITY_ID)
```

Comment:
```suggestion
	if (Connection != nullptr)
```

Reply:
If you refresh you will see that's gone.